### DISCUSSION TITLE
https://plume.pengzhanbo.cn/guide/quick-start/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#82] [docs-comment]
# https://plume.pengzhanbo.cn/guide/quick-start/

依赖环境 Node.js v18.16.0+ pnpm 8+ 或 Yarn 2+ 提示 使用 pnpm 时，你需要安装 vue 作为 peer-dependencies 。 使用 Yarn 2+ 时，你需要在 .yarnrc.yml 文件中设置 nodeLinker: 'node-modules' 。 安装 使用本主题，你需要首先新建一个项目，并安装v...

https://plume.pengzhanbo.cn/guide/quick-start/

**Comment by tourcoding** [comment]
为什么安装了vuepress-plugin-md-enhance，博文使用mermaid  echarts也没问题，但是执行pnpm dev的时候还会提示vuepress-plugin-md-enhance:  ✖ Package chart.js is not installed, please install it manually!      warning plugin vuepress-plugin-md-enhance has been used multiple times, only the last one will take effect

**Reply by pengzhanbo** [reply]
不需要重复安装 `vuepress-plugin-md-enhance`，主题已经内置了这个插件，只需要在 `plugin.markdownEnhance` 中启用相关功能即可。查看[图表 chart.js](https://plume.pengzhanbo.cn/guide/chart/chartjs/#%E9%85%8D%E7%BD%AE) 文档。
你需要做的是在你的项目中手动安装 `chart.js`。因为图表相关的功能其依赖的包相对会比较大，所以就没有内置安装在主题中。需要自行选择想要使用 图表库，然后安装它。
